presto-spi/src/main/java/com/facebook/presto/spi/ConnectorSplit.java

```diff
@@ -13,7 +13,14 @@
  */
 package com.facebook.presto.spi;
 
+import com.facebook.presto.spi.schedule.NodeSelectionStrategy;
+
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+
+import static com.facebook.presto.spi.StandardErrorCode.NODE_SELECTION_STATEGY_NOT_SUPPORTED;
 
 public interface ConnectorSplit
```

Comment:
Remove `isRemotelyAccessible` interface; replace it with `NodeSelectionStrategy = HARD_AFFINITY `;